spec/lib/simp/beaker_helpers_spec.rb

```diff
@@ -103,20 +103,17 @@ def initialize
     end
 
     it 'uses defaults when no environment variables are set' do
-      expected = {
```

Comment:
Shouldn't we add pup6 tests to this? (in place of the pup4 tests)

Reply:
Pup6 and Pup5 work the same way. We can fix this when we decide on pulling Puppet 4 or not as a later ticket.